mathics/builtin/distance/numeric.py

```diff
@@ -22,7 +23,7 @@
 
 
 def _norm_calc(head, u, v, evaluation):
```

Comment:
Tests seem to indicate that head is sometimes a symbol and sometimes a string. 

One reason I like to have type annotations in there (e.g. for `head`).

Reply:
Yep... working on it.